sdk/storage/storage-queue/src/utils/tracing.ts

```diff
@@ -17,11 +17,10 @@ export function createSpan(
   const tracer = getTracer();
   const spanOptions: SpanOptions = {
     ...tracingOptions.spanOptions,
-    kind: SpanKind.CLIENT
+    kind: SpanKind.INTERNAL
   };
 
   const span = tracer.startSpan(`Azure.Storage.Queue.${operationName}`, spanOptions);
-  span.setAttribute("component", "storage");
```

Comment:
Are these being set automatically somewhere or just no longer needed?

Reply:
According to @lmolkova "We are still discussing what should be there across all SDKs."

Reply:
Is this a breaking change?

Reply:
Tracing is considered pre-release still (as core-tracing is still in preview) so things like tracing metadata and events is not considered GA.